resilience4j-retrofit/README.adoc

```diff
@@ -0,0 +1,39 @@
+= Retrofit Circuitbreaker
+
+https://square.github.io/retrofit/[Retrofit] client circuit breaking
+using
+https://github.com/robwin/javaslang-circuitbreaker[Javaslang-circuitbreaker]
+
```

Comment:
Copied the `retrofit.adoc` page back to the module `README.adoc`  - the module readme also has license block as found elsewhere.